okhttp-tests/src/test/java/okhttp3/EventListenerTest.java

```diff
@@ -57,6 +58,7 @@
 import org.junit.Test;
 
 import static java.util.Arrays.asList;
+import static okhttp3.RecordingEventListener.*;
```

Comment:
Star imports are forbidden. Ensure you're using https://github.com/square/java-code-styles

Reply:
Yep, not sure why it happened.

![image](https://user-images.githubusercontent.com/231923/29892635-3c22e47e-8dc7-11e7-86b6-c1acfae1f753.png)

Reply:
Seems like an intellij bug with 
![image](https://user-images.githubusercontent.com/231923/29892709-99f4b2f8-8dc7-11e7-977c-a2af9ec94902.png)

Reply:
TIL: inner classes can be imported with static or normal imports...